manim/mobject/svg/text_mobject.py

```diff
@@ -87,105 +121,58 @@ def __init__(self, text, **config):
                     each.add_line_to(last)
                     last = points[index + 1]
             each.add_line_to(last)
-
         if self.t2c:
             self.set_color_by_t2c()
         if self.gradient:
             self.set_color_by_gradient(*self.gradient)
         if self.t2g:
             self.set_color_by_t2g()
-
         # anti-aliasing
         if self.height is None and self.width is None:
             self.scale(TEXT_MOB_SCALE_FACTOR)
 
-    def get_space_width(self):
+    """
+    def get_extra_space_perc(self):
```

Comment:
What's this? A commented out function?

Reply:
@NavpreetDevpuri let's figure this one out and leave the rest of the documentation for a future PR.

Reply:
very old function from an old implementation just removed it.